modules/doi/includes/utilities.inc

```diff
@@ -864,3 +833,38 @@ function islandora_doi_get_mods($id) {
 
   return islandora_doi_translate($source_string, $content_type);
 }
+
+/**
+ * Filter HTML tags from imported XML metadata.
+ *
+ * @param string $text
+ *   The text of the metadata to be filtered 
+ * @param array $removals
+ *   An array of strings to be removed from $text prior to filtering
+ *
+ * @return string 
+ *   String filtered using value of $islandora_doi_handle_improted_metadata 
+ */
+function islandora_doi_filter_html($text, $removals = FALSE) {
+  if ($removals) {
+    foreach ($removals as $removal) {
+      $text = str_replace($removal, '', $text);
```

Comment:
I guess we should start by testing for `$text` being `FALSE`, since [`DOMDocument::saveXML()`](http://php.net/manual/en/domdocument.savexml.php) might return that on error... Perhaps we could return an empty string and/or issue a warning in that case?

Reply:
Better safe than sorry. Adding now.